osbrain/agent.py

```diff
@@ -1548,7 +1598,7 @@ def send_recv(self, address, message):
     @Pyro4.oneway
     def run(self):
         """
-        Run the agent.
+        Begin execution of the agent process and start the main loop.
```

Comment:
Not really the begin of the agent process execution. Running the agent means starting the main loop, that is correct though.